mod.config

```diff
@@ -9,7 +9,7 @@
 MOD_ID="tda"
 
 # The OpenRA engine version to use for this project.
-ENGINE_VERSION="release-20180307"
+ENGINE_VERSION="release-20180729"
```

Comment:
This should be `ENGINE_VERSION="playtest-20180729"` (playtest instead of release).

Reply:
oh man i feel so stupid. Thank you for spoting that aswell man!